packages/react-reconciler/src/ReactFiberReconciler.js

```diff
@@ -401,5 +422,22 @@ export default function<T, P, I, TI, PI, C, CC, CX, PL>(
       }
       return hostFiber.stateNode;
     },
+
+    injectIntoDevTools(devToolsConfig: DevToolsConfig<I, TI>): boolean {
+      const {findFiberByHostInstance} = devToolsConfig;
+      return ReactFiberDevToolsHook.injectInternals({
+        ...devToolsConfig,
+        findHostInstanceByFiber(fiber: Fiber): I | TI | null {
+          return findHostInstance(fiber);
+        },
+        findFiberByHostInstance(instance: I | TI): Fiber | null {
+          if (!findFiberByHostInstance) {
+            // Might not be implemented by the renderer.
+            return null;
+          }
+          return findFiberByHostInstance(instance);
+        },
+      });
```

Comment:
Are the wrapper functions necessary? Couldn't we just do...
```js
    injectIntoDevTools(devToolsConfig: DevToolsConfig<I, TI>): boolean {
      const {findFiberByHostInstance} = devToolsConfig;
      const findFiberByHostInstanceImpl = findFiberByHostInstance
        ? findFiberByHostInstance
        : () => null;

      return ReactFiberDevToolsHook.injectInternals({
        ...devToolsConfig,
        findHostInstanceByFiber: findHostInstance,
        findFiberByHostInstance: findFiberByHostInstanceImpl,
      });
    },
```

Reply:
That would work too. I don't feel strongly as they're only called once.

Reply:
I tried using your approach, but it seems like Flow fails to catch some issues for some reason now. I'd prefer keeping it explicit so it's better typed.

Reply:
Boo, Flow.

Okay. No biggie.